### PR TITLE
Add uopz 6.x functions

### PR DESCRIPTION
**Pull request by zonuexe**
refs https://github.com/krakjoe/uopz/blob/v6.1.2/uopz.c

The parameters in the [PHP manual](https://www.php.net/manual/en/function.uopz-add-function.php) are different from the [implementation](https://github.com/krakjoe/uopz/blob/v6.1.2/uopz.c).

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Thank you!

File: resources/functionMap.php [hunk 1]
```diff
@@ -12668,11 +12678,19 @@
...
 'uopz_restore\'1' => ['void', 'function'=>'string'],
+'uopz_set_hook' => ['bool', 'class'=>'string', 'class'=>'function', 'hook'=>'Closure'],
+'uopz_set_hook\1' => ['bool', 'class'=>'string', 'hook'=>'Closure'],
 'uopz_set_mock' => ['void', 'class'=>'string', 'mock'=>'object|string'],
+'uopz_set_property' => ['void', 'class'=>'string', 'property'=>'string', 'value'=>'mixed'],
+'uopz_set_property\1' => ['void', 'instance'=>'object', 'property'=>'string', 'value'=>'mixed'],
 'uopz_set_return' => ['bool', 'class'=>'string', 'function'=>'string', 'value'=>'mixed', 'execute='=>'bool'],
 'uopz_set_return\'1' => ['bool', 'function'=>'string', 'value'=>'mixed', 'execute='=>'bool'],
+'uopz_set_return' => ['bool', 'class'=>'string', 'function'=>'string', 'static'=>'array'],
+'uopz_set_return\'1' => ['bool', 'function'=>'string', 'static='=>'array'],
```
Comment (SMillerDev) [inline resources/functionMap.php:12689]: I might be misunderstanding this, but currently PHPStan is complaining that `Parameter #3 $static of function uopz_set_return expects array` while parameter #3 is actually the value in the following form:
https://github.com/krakjoe/uopz/blob/v6.1.2/uopz.c#L185-L186
```php
 uopz_set_return ( string $class , string $function , mixed $value , bool $execute = false ) : bool
```

**Comment by SMillerDev** [comment]
@ondrejmirtes I don't think this does what it's supposed to actually.

File: resources/functionMap.php (hunk 1 above)
Comment (ondrejmirtes) [reply resources/functionMap.php:12689]: @SMillerDev Hi, this is a pretty old PR. If you see a buggy behaviour, please open an issue with reproduction, thank you.

Comment (SMillerDev) [reply resources/functionMap.php:12689]: Will do, thanks.

**Pull request merged**